python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
```

Comment:
In my opinion this file is too complicated to justify the use of `from <foo> import *`. It is not just a case of "hoist a few names into an `<foo>Lib.py` file so the `__init__.py` file can hoist them yet another level up".

I am very reluctantly willing to accept trivial cases (though even those violate our standards and should trigger an RFC), but I'm really unhappy about it in any file that has nontrivial code.